bubble_sort/Python/bubble_sort.py

```diff
@@ -0,0 +1,14 @@
+
+def bubbleSort(arr):
+    n = len(arr)
+    for i in range(n):
+        for j in range(0, n-i-1):
+            if arr[j] > arr[j+1]:
+                arr[j], arr[j+1] = arr[j+1], arr[j]
+
+a = [76,74,63,74,24,93,25]
+
+bubbleSort(a)
+
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmphheo796q/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmphheo796q/bubble_sort/Python/bubble_sort.py
@@ -10,5 +10,5 @@
 
 bubbleSort(a)
 
-print ("Sorted array is:")
+print("Sorted array is:")
 print(a)
```